app/components/concept-page/question-card-option.ts

```diff
@@ -1,6 +1,20 @@
 import Component from '@glimmer/component';
 
-export default class QuestionCardOptionComponent extends Component {
+export interface Signature {
+  Element: HTMLButtonElement;
+
+  Args: {
+    option: {
+      is_correct: boolean;
+      isSelected: boolean;
```

Comment:
There's a mixture of camelCase and snake_case here. Any way we can avoid this?

Reply:
At the very least let's add a comment here - I know that this is old code, but we didn't intend to have this kind of mixture of casing

Reply:
Also, let's try to import `Option` from `ConceptQuestion` model rather than re-defining it here?